android-libkiwixbuild/app/src/main/cpp/CMakeLists.txt

```diff
@@ -30,23 +20,17 @@ add_library(
         kiwixserver.cpp
         kiwixicu.cpp
 )
-#[[add_library(kiwix SHARED IMPORTED)
 
-set_property(TARGET kiwix PROPERTY IMPORTED_LOCATION /home/hp-pc/Documents/android-publisher/android-libkiwixbuild/app/src/main/jniLibs/${ANDROID_ABI}/libkiwix.so)
-add_library(zim SHARED IMPORTED)
-
-set_property(TARGET zim PROPERTY IMPORTED_LOCATION /home/hp-pc/Documents/android-publisher/android-libkiwixbuild/app/src/main/jniLibs/${ANDROID_ABI}/libzim.so)]]
-#
 find_library(libkiwix
-        kiwix # Lib's full name in this case - libfoo.so
```

Comment:
The comment you've removed state that we must use the full name of the library (`libkiwix.so`). But you have use `kiwix` only. Why ? Is it working with `kiwix` ?

Reply:
Actually its work like this if the file name is ```libfoo.so``` then library name will be ```foo```, in our case file name is ```libkiwix.so``` so its imported as ```kiwix``` and yes its working .

Reply:
Ho, I miss read the comment.
It states that if name is kiwix, full name will be libkiwix.so. Not that we must write libkiwix.so.
My bad.